### PR TITLE
docs browser: Add entry for string.byte function

### PR DESCRIPTION
**Pull request by gibran-padilla**
### Description

New term entry for the string.byte() function of lua programming language.
Includes a breve description, syntax explanation and examples.

### Type of Change

New entry to docs browser

### Checklist

- [✓ ] All writings are my own.
- [ ✓] My entry follows the Codecademy Docs style guide.
- [✓ ] My changes generate no new warnings.
- [✓ ] I have performed a self-review of my own writing and code.
- [✓ ] I have checked my entry and corrected any misspellings.
- [✓ ] I have made corresponding changes to the documentation if needed.
- [✓ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✓ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+Title: 'byte()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'string.byte is a function that returns the ASCII (decimal) equivalent of a character.' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'computer-science'
+
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:6]:
```suggestion
 - 'Game Development'
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Strings'
+  - 'Characters'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-example-course'
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:11]:
```suggestion
  - 'learn-lua'
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  - 'paths/example-path'
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:12]:
```suggestion
  - 'paths/computer-science'
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+**String.byte()** function takes a character or a string as an argument and returns an ASCII (decimal) equivalent internal numeric representation.
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:15]:
```suggestion
The **`string.byte()`** function takes a character or a string as an argument and returns an ASCII (decimal) equivalent internal numeric representation.
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## Syntax
+
+The Function string.byte(s, n) can take two arguments, where s parameter is a character or string and n is the index of the string passed as an argument. Passing only one argument will set n to default 1.
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:19]:
```suggestion
```pseudo
string.byte(s, n)
```

The function can take two arguments, where the `s` parameter is a character or string and `n` is the index of the string passed as an argument. Passing only one argument will set `n` to `1`.
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## Arguments
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:21]:
```suggestion
## Example 1
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+The function first argument can take a string or digit  but a symbol will throw an error.
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:23]:
```suggestion
The example below demonstrates the use of the `.byte()` method.
```

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+```lua
+print(string.byte('a')) -- Prints: 97
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:26]: typically the output is shown in a separate ```shell block

File: content/lua/concepts/strings/terms/byte/byte.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+print(string.byte('9')) -- Prints: 57
+print(string.byte(9))   -- Prints: 57
+print(string.byte(' ')) -- Prints: 32
+print(string.byte(';')) -- Throws error
+```
+
+Adding a second argument separated by acomma.
```
Comment (caupolicandiaz) [inline content/lua/concepts/strings/terms/byte/byte.md:33]:
```suggestion
## Example 2

The following code demonstrates the use of this function with a second argument.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@gibran-padilla I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by gibran-padilla** [comment]
@caupolicandiaz Thank you very much for the review. 

I made the changes suggested, so I would be happy if you can check the new entry.

Regards.

**Review by caupolicandiaz — APPROVED** [review]
@gibran-padilla Some of the edits were not implemented correctly, so I went ahead and made them directly. Please review the changes for future reference. This is ready for a second review.

**Comment by gibran-padilla** [comment]
@caupolicandiaz I appreciate your comments, 
I will try to improve the mistake made this time.

Regrads.

**Review by SSwiniarski — APPROVED** [review]
@gibran-padilla I made one small edit and it looks ready to merge.

**Pull request merged**